src/test/scala/mesosphere/marathon/core/health/impl/MarathonHealthCheckManagerTest.scala

```diff
@@ -52,27 +51,15 @@ class MarathonHealthCheckManagerTest extends AkkaUnitTest with Eventually {
     )
   }
 
-  def makeRunningTask(appId: PathId, version: Timestamp)(implicit instanceTracker: InstanceTracker): (Instance.Id, Task.Id) = {
-    val instance = Instance.scheduled(AppDefinition(appId, versionInfo = VersionInfo.forNewConfig(version)))
-    val (taskId, _) = instance.tasksMap.head
-    val taskStatus = TestTaskBuilder.Helper.runningTask(instance.instanceId).status.mesosStatus.get
-
-    instanceTracker.process(Provision(instance)).futureValue
-    instanceTracker.updateStatus(instance, taskStatus, clock.now()).futureValue
-
-    (instance.instanceId, taskId)
-  }
-
   def setupTrackerWithProvisionedInstance(appId: PathId, version: Timestamp, instanceTracker: InstanceTracker): Future[Instance] = async {
     val app = AppDefinition(appId, versionInfo = VersionInfo.OnlyVersion(version))
     val scheduledInstance = Instance.scheduled(app)
     // schedule
     await(instanceTracker.schedule(scheduledInstance))
     // provision
-    val provisionedInstance = scheduledInstance.provisioned(AgentInfoPlaceholder(), NetworkInfoPlaceholder(), app, Timestamp.now(), Task.Id.forInstanceId(scheduledInstance.instanceId))
-    await(instanceTracker.process(InstanceUpdateOperation.Provision(provisionedInstance)))
+    await(instanceTracker.process(InstanceUpdateOperation.Provision(scheduledInstance.instanceId, AgentInfoPlaceholder(), version, Seq(Task.provisioned(Task.Id.forInstanceId(scheduledInstance.instanceId), NetworkInfoPlaceholder(), version, Timestamp.now())), Timestamp.now())))
```

Comment:
for some reason, provision was always called via `.process()`. I would defer refactoring this until we have some idea how we want the scheduler interface to look like so that we don't change it multiple times

I extracted the `Timestamp.now()`